api/proto/teleport/legacy/types/types.proto

```diff
@@ -505,6 +510,16 @@ message RedshiftServerless {
   string WorkgroupID = 3 [(gogoproto.jsontag) = "workgroup_id,omitempty"];
 }
 
+// OpenSearch contains AWS OpenSearch specific metadata.
+message OpenSearch {
+  // DomainName is the name of the domain.
+  string DomainName = 1 [(gogoproto.jsontag) = "domain_name,omitempty"];
+  // DomainID is the ID of the domain.
+  string DomainID = 2 [(gogoproto.jsontag) = "domain_id,omitempty"];
+  // EndpointType is the type of the endpoint.
+  string EndpointType = 3 [(gogoproto.jsontag) = "endpoint_type,omitempty"];
+}
```

Comment:
Where are these values used? I don't mind keeping them but also seems unnecessary.

Reply:
They are not used in the codebase right now, but these can be useful both in the future and for customer-side scripting.